docs/misc/deprecated.md

```diff
@@ -12,13 +12,23 @@ parent = "mn_use_docker"
 
 The following list of features are deprecated.
 
+### Command line short form options
+**Deprecated In Release: v1.9**
+
+**Target For Removal In Release: v1.11**
+
+The following single-dash options are deprecated:
+
+    docker run -c
+    docker run -m
```

Comment:
Perhaps mention that the long variants (`--memory`, `--cpu-shares`) should be used instead